src/scripts/ping-okta-server.ts

```diff
@@ -0,0 +1,117 @@
+import fetch from 'node-fetch';
+import * as TE from 'fp-ts/lib/TaskEither';
+import * as E from 'fp-ts/lib/Either';
+import { RootCommand } from '..';
+import * as Console from 'fp-ts/lib/Console';
+import { pipe } from 'fp-ts/lib/function';
+import { Argv } from 'yargs';
+
+export const pingOktaServer = async (
+  clientId: string,
+  organisationUrl: string
+): Promise<TE.TaskEither<Error, string>> => {
+  const response = await TE.chain(
+    ({
+      organisationUrl,
+      clientId,
+    }: {
+      readonly organisationUrl: string;
+      readonly clientId: string;
+    }) =>
+      TE.tryCatch(
+        async () => {
+          return await fetch(
+            `${organisationUrl}/oauth2/default/.well-known/oauth-authorization-server?client_id=${clientId}`
+          );
+        },
+        (error: unknown) => {
+          // eslint-disable-next-line functional/no-conditional-statement
+          if (error instanceof Error) {
+            return error;
+          }
+          return new Error('Failed to list users.', {
+            cause: error,
+          });
+        }
+      )
+  )(TE.right({ organisationUrl, clientId }))();
+
+  // eslint-disable-next-line functional/no-conditional-statement
+  if (E.isLeft(response)) {
+    return TE.left(response.left);
+  }
+  const rightResponse = response.right;
+  // eslint-disable-next-line functional/no-conditional-statement
+  if (rightResponse.status >= 500) {
+    // eslint-disable-next-line functional/no-conditional-statement
+    return TE.left(
+      new Error('Server error. Please wait and try again later.', {
+        cause: rightResponse,
+      })
+    );
+  }
+  // eslint-disable-next-line functional/no-conditional-statement
+  if (rightResponse.status >= 400 && rightResponse.status < 500) {
+    return TE.left(
+      new Error(
+        'Client error. Please check your client id and the URL of your organisation.',
+        {
+          cause: rightResponse,
+        }
+      )
+    );
+  }
+  // eslint-disable-next-line functional/no-conditional-statement
+  if (rightResponse.status >= 200 && rightResponse.status < 300) {
+    return TE.right('Okta server is up and running.');
+  }
+  return TE.left(
+    new Error('Unexpected response', {
+      cause: rightResponse,
+    })
+  );
+};
+
+export default (
+  // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+  rootCommand: RootCommand
+): Argv<{
+  readonly clientId: string;
+  readonly privateKey: string;
+  readonly organisationUrl: string;
+  readonly groupId?: string;
+}> =>
+  rootCommand.command(
+    'list-users2',
```

Comment:
**nit:** change the command name and description.

Reply:
modified in commit: 9f846a4fc822510fa69e963a39aa76a76ddec247